.vscode/settings.json

```diff
@@ -1,13 +1,10 @@
 {
-  "cSpell.words": ["httpx", "nextra"],
+  "cSpell.words": ["httpx", "nextra", "toolkits"],
   "editor.wordWrap": "bounded",
   "editor.wordWrapColumn": 80,
-  "editor.formatOnSave": true,
-  "[javascript][typescript][json][jsonc][mdx][tsx][jsx][css]": {
+  "editor.formatOnSave": false,
```

Comment:
Turned off format-on-save so that it stops messing up our mdx files. (hopefully we can figure out why later, and turn it back on)